src/runner/browser-env/vite/browser-modules/mocha/index.ts

```diff
@@ -33,6 +34,8 @@ export class MochaWrapper {
         }
 
         this._socket.emit(BrowserEventNames.initialize, getErrorsOnPageLoad(error));
+
+        wrapConsoleMethods();
```

Comment:
Do it before run any runnable. I do not call it earlier so that all sorts of browser system logs are not displayed